commands/serve/web/render-fixture.js

```diff
@@ -0,0 +1,97 @@
+import { embed } from '@nebula.js/stardust';
+import EnigmaMocker from '@nebula.js/enigma-mocker';
+import extend from 'extend';
+import { info as getServerInfo } from './connect';
+import { getModule } from './hot';
+
+const getServeOptions = async ({ themes = [], supernova, flags }) => {
+  // load js artifact provided as entry
+  const mo = await getModule(supernova.name, supernova.url);
+
+  return {
+    type: supernova.name,
+    load: async () => mo,
+    instanceConfig: {
+      themes: themes.map((t) => ({
+        id: t,
+        load: async () => (await fetch(`/theme/${t}`)).json(),
+      })),
+      context: {
+        constraints: {},
+      },
+      flags,
+    },
+  };
+};
+
+const getUrlParamOptions = (params) => ({
+  instanceConfig: {
+    context: {
+      theme: params.theme,
+      language: params.language,
+    },
+  },
+});
+
+// Priority: serve options -> fixture -> URL params
```

Comment:
Is this documented somewhere else? Think people can interpret this in different ways.

Reply:
Funny thing. It was written the other way around about 10 minutes before I submitted the PR. However I changed it in the last minute in order to align with the the order of how `extend(true, {}, obj1, obj2, ...)` works. I think it's best explained in the code as it leaves little room for interpretation so I removed this comment.